src/axom/primal/operators/detail/in_curved_polygon_impl.hpp

```diff
@@ -110,12 +147,11 @@ double convex_endpoint_winding_number(bool is_init, const BezierCurve<T, 2>& c)
  *
  * \param [in] q The query point at which to compute winding number
  * \param [in] c The BezierCurve object along which to compute the winding number
- * \param [in] convex_cp Boolean flag if the input Bezier curve is already convex
- * \param [in] linear_tol The tolerance level at which a BezierCurve object is to
- *             be interpreted as linear.
- * \param [in] edge_tol The tolerance level at which we consider a query point
- *             to be exactly on the Bezier curve
- *
+ * \param [in] isConvex Boolean flag if the input Bezier curve is already convex
+ * \param [in] edge_tol The physical distance level which objects are indistinguishable
+ * \param [in] EPS Miscellaneous numerical tolerance for nonphysical distances, used in
+ *   isLinear, isNearlyZero, in_polygon, is_convex
```

Comment:
In my opinion, `isLinear()` should stay camel case since it's a method of `BezierCurve`, but the rest should all have pot hole style. When `isLinear()` is (eventually) moved to an independent operator, I would expect it to become `is_linear()`. I don't have control over what `axom::utilities::isNearlyEqual` is called unfortunately, but in my opinion it shouldn't be camel case.